src/main/java/com/parasoft/findings/jenkins/coverage/ParasoftCoverageQualityGate.java

```diff
@@ -0,0 +1,76 @@
+package com.parasoft.findings.jenkins.coverage;
+
+import edu.hm.hafner.coverage.Metric;
+import hudson.Extension;
+import hudson.model.AbstractProject;
+import hudson.model.Item;
+import hudson.util.ListBoxModel;
+import io.jenkins.plugins.coverage.metrics.model.Baseline;
+import io.jenkins.plugins.coverage.metrics.model.ElementFormatter;
+import io.jenkins.plugins.util.JenkinsFacade;
+import io.jenkins.plugins.util.QualityGate;
+import org.kohsuke.stapler.AncestorInPath;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+import org.kohsuke.stapler.verb.POST;
+
+public class ParasoftCoverageQualityGate extends QualityGate {
```

Comment:
Add a `serialVersionUID` field like `CoverageQualityGate`, in case there's problem related to serialization.